### PR TITLE
PHOENIX-6639 Read repair for transforming table

### PR DESCRIPTION
**Pull request by gokceni**

### CODE REVIEW

**Comment by gokceni** [comment]
@gjacoby126 @tkhurana

**Review by gjacoby126 — CHANGES_REQUESTED** [review]
Looks good, just some minor stuff

File: phoenix-core/src/main/java/org/apache/phoenix/iterate/TableResultIterator.java [hunk 1]
```diff
@@ -140,6 +141,23 @@ public TableResultIterator(MutationState mutationState, Scan scan, ScanMetricsHo
         this.retry=plan.getContext().getConnection().getQueryServices().getProps()
                 .getInt(QueryConstants.HASH_JOIN_CACHE_RETRIES, QueryConstants.DEFAULT_HASH_JOIN_CACHE_RETRIES);
         ScanUtil.setScanAttributesForClient(scan, table, plan.getContext().getConnection());
+
+        ScanUtil.setScanAttributesForIndexReadRepair(scan, table, plan.getContext().getConnection());
```
[the remark this answers is not in the document]
Comment (gokceni) [reply phoenix-core/src/main/java/org/apache/phoenix/iterate/TableResultIterator.java:145]: Good catch! They are the same.

**Review by gjacoby126 — APPROVED** [review]
+1, thanks @gokceni

**Comment by gokceni** [comment]
Merged.

**Pull request closed without merging**